test/validators.js

```diff
@@ -5513,7 +5513,7 @@ describe('Validators', () => {
         '2009123',
         '2009-05',
         '2009-123',
-        '2009-222',
+        // '2009-222', // invalid test-case, implies 2009-22-2
```

Comment:
@chriso - I also found 2 invalid test cases in the cause of raising this PR, which I commented out. Let me know if this looks okay with you.

Reply:
This is a valid [ordinal date](https://en.wikipedia.org/wiki/ISO_8601#Ordinal_dates). The 222 indicates that it's 222 days into the year.